doc/content/getting_started/RunningBlackBear.md

```diff
@@ -72,17 +81,17 @@ cd ~/projects/blackbear/
 git pull
 ```
 
-Then update the MOOSE submodule:
+Then update MOOSE:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update
+cd ~/projects/moose/
+git pull
 ```
 
```

Comment:
Would it be prudent to include some information about git clean before rebuilding libmesh?

Reply:
I'd rather keep this simple. That isn't strictly necessary -- I never do it. They could accidentally delete files they care about if they do a `git clean1.